package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "jsonschema": "1.1.1",
-    "lodash": "4.17.10"
+    "lodash": "^4.17.11"
```

Comment:
I'd remove the `^` as it's important that we know exactly what version of lodash goes out. not that I expect lodash to break anything or do anything weird, but it ensures we're in control of what goes out.

Reply:
IMHO that's what you have a lock-file for, but I'll drop it :)